### PR TITLE
ConfigurationOverConvention - specifying calling convention explicitly

### PR DESCRIPTION
**Pull request by moozzyk**
on public API

### CODE REVIEW

File: include/signalrclient/connection.h [hunk 1]
```diff
@@ -19,8 +19,8 @@ namespace signalr
 
     class connection
     {
-        // TODO: consider making public or remove?
-        typedef std::function<void(const utility::string_t&)> message_received;
+    public:
+        typedef std::function<void __cdecl (const utility::string_t&)> message_received_handler;
 
     public:
```
Comment (BrennanConroy) [inline include/signalrclient/connection.h]: remove this `public:` now?

File: include/signalrclient/hub_proxy.h [hunk 2]
```diff
@@ -18,34 +18,36 @@ namespace signalr
     class hub_proxy
     {
     public:
+        typedef std::function<void __cdecl (const web::json::value&)> method_invoked_handler;
+        typedef std::function<void __cdecl (const web::json::value&)> on_progress_handler;
+
+    public:
```
Comment (BrennanConroy) [inline include/signalrclient/hub_proxy.h]: Remove this one too?

File: include/signalrclient/connection.h (hunk 1 above)
Comment (moozzyk) [inline include/signalrclient/connection.h]: I wanted to have `typedef`s in their own section but it does not really matter too much. Can remove.

File: include/signalrclient/hub_proxy.h (hunk 2 above)
Comment (moozzyk) [inline include/signalrclient/hub_proxy.h]: OK

**Comment by BrennanConroy** [comment]
:shipit:

**Comment by moozzyk** [comment]
Thanks @BrennanConroy !

**Pull request merged**